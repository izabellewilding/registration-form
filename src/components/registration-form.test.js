import React from "react";
import { render, cleanup } from "@testing-library/react";
import RegistrationForm from "./registration-form";

it("renders", () => {
  const { asFragment } = render(<RegistrationForm />);
  expect(asFragment()).toMatchSnapshot();
});
