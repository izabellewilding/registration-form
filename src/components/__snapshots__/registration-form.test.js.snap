// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm sc-AxirZ cjPigA"
  >
    <h1
      class="sc-AxhUy djFllI"
    >
      Create an Account
    </h1>
    <div
      class="sc-AxjAm sc-AxiKw cZcByZ"
    >
      <p
        class="sc-AxgMl eKgqRt"
      >
        Sign up with your name, email, and a password.
      </p>
      <form
        class="sc-AxhCb buIdnp"
        method="POST"
      >
        <div
          class="sc-fzozJi kenODZ"
        >
          <p
            class="sc-AxgMl sc-AxmLO eIjKmR"
            for="name"
          >
            <svg>
              user.svg
            </svg>
          </p>
          <input
            class="sc-AxheI lhaela"
            name="name"
            placeholder="Name"
            required=""
            type="text"
            value=""
          />
        </div>
        <div
          class="sc-fzozJi kenODZ"
        >
          <p
            class="sc-AxgMl sc-AxmLO eIjKmR"
            for="email"
          >
            <svg>
              email.svg
            </svg>
          </p>
          <input
            class="sc-AxheI lhaela"
            name="Email"
            placeholder="Email"
            required=""
            type="text"
            value=""
          />
        </div>
        <div
          class="sc-fzozJi kenODZ"
        >
          <p
            class="sc-AxgMl sc-AxmLO eIjKmR"
            for="password"
          >
            <svg>
              padlock.svg
            </svg>
          </p>
          <input
            class="sc-AxheI sc-Axmtr jUZfFS"
            name="password"
            pattern="^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$"
            placeholder="Password"
            required=""
            type="password"
            value=""
          />
        </div>
        <div
          class="sc-fzozJi kenODZ"
        >
          <p
            class="sc-AxgMl sc-AxmLO eIjKmR"
            for="confirmPassword"
          >
            <svg>
              padlock.svg
            </svg>
          </p>
        </div>
        <input
          class="sc-AxheI sc-Axmtr jUZfFS"
          name="confirmPassword"
          placeholder="Confirm password"
          required=""
          type="password"
          value=""
        />
        <button
          class="sc-fzoLsD gjkvof"
          type="submit"
        >
          <p
            class="sc-AxgMl bFBayz"
          >
            Submit
          </p>
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
